bundles/org.eclipse.cdt.lsp.clangd/src/org/eclipse/cdt/lsp/clangd/ClangdConfigurationManager.java

```diff
@@ -0,0 +1,191 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Bachmann electronic GmbH and others.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ * Gesa Hentschke (Bachmann electronic GmbH) - initial implementation
+ *******************************************************************************/
+
+package org.eclipse.cdt.lsp.clangd;
```

Comment:
Usually the component is a part of implementation, and not a part of [potential] API, it should be placed to `internal` package

Reply:
I've to rework this part as I stumped about this https://docs.osgi.org/specification/osgi.cmpn/7.0.0/service.component.html#:~:text=A%20bundle%20containing,components%20being%20activated.

Reply:
Please take a look at my last commit